### PR TITLE
metadata for scratch

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 1]
```diff
@@ -390,6 +395,14 @@ def _write_demultiplexed_logs(self, build_id, logs_dir):
                 prefix = 'orchestrator' if platform is None else platform
                 log_filename = os.path.join(logs_dir, "%s.log" % prefix)
                 platform_logs[platform] = open(log_filename, 'wb')
+
+            if re.search(SCRATCH_METADATA_REGEX, entry.line):
+                meta_file = entry.line.split("'")[1]
```
Comment (MartinBasti) [inline koji_containerbuild/plugins/builder_containerbuild.py]: I've no idea how entry item looks, do you have some examples?

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 2]
```diff
@@ -97,6 +99,9 @@
 }
 
 
+SCRATCH_METADATA_REGEX = ".*uploaded.*metadata.json'$"
```
Comment (athos-ribeiro) [inline koji_containerbuild/plugins/builder_containerbuild.py]: You can skip the '.' as '\.json'.

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 3]
```diff
@@ -386,6 +390,14 @@ def _write_demultiplexed_logs(self, build_id, logs_dir):
         platform_logs = {}
         for entry in logs:
             platform = entry.platform
+            if platform == METADATA_TAG:
```
[the remark this answers is not in the document]
Comment (rcerven) [reply koji_containerbuild/plugins/builder_containerbuild.py]: not really

**Comment by rcerven** [comment]
rn updated: - move metadata.json to task output for scratch builds

**Pull request merged**